### ISSUE TITLE
Update notification about accepted PRs after merge

### ISSUE THREAD

**Issue by ad-m-ss** [integrations/slack#1700]
**Is your feature request related to a problem? Please describe.**

I would like to improve the reading of notifications to update the ones which are no longer relevant.

I would like leverage crossing out the notification when it's get expired. For example there is notifications which is crossed out, because I did review:

<img width="494" alt="image" src="https://github.com/integrations/slack/assets/99484706/906175a3-6781-4e57-8c16-12385149ce21">

**Describe the solution you'd like**

I would like a cross-out notification about approved PRs when PRs are merged. I would like to update that notification:

<img width="419" alt="image" src="https://github.com/integrations/slack/assets/99484706/f8f9fddb-7879-477e-bb33-42b952e88e4d">

Once PRs is merged, I think it's safe to assume that it's done, so all notification about that PRs can be cross-out.

**Describe alternatives you've considered**

We may also delete expired notifications, but I think it will be inconsistent and may be confusing.
